plaso/output/elastic.py

```diff
@@ -13,7 +13,70 @@ class ElasticsearchOutputModule(shared_elastic.SharedElasticsearchOutputModule):
 
   NAME = 'elastic'
   DESCRIPTION = 'Saves the events into an Elasticsearch database.'
-
+  DEFAULT_MAPPING = {
+      'properties': {
```

Comment:
can you make sure that all quotes are single (') instead of double ("), to be consistent with the rest of the code base

Reply:
can you resolve this one